src/Admin.ts

```diff
@@ -90,6 +92,13 @@ export default class Admin {
                 adminChannel = await guild.channels.create({name: this.sharedSettings.server.guruChannel, type: Discord.ChannelType.GuildText});
             }
         }
+        if (!adminLogChannel) {
+            try {
+                adminLogChannel = await guild.channels.create({name: this.sharedSettings.server.guruLogChannel, type: Discord.ChannelType.GuildText});
```

Comment:
Needs `this.sharedSettings.botty.isProduction` check (see line 87)

Reply:
Done